app/src/main/java/com/firebase/firebaseui_android/DemoApplication.java

```diff
@@ -0,0 +1,11 @@
+package com.firebase.firebaseui_android;
```

Comment:
The main classes are in com.firebase.ui. I'd rather have this in the same package or something along the same lines.